dist/spec.yaml

```diff
@@ -10204,6 +10204,17 @@ definitions:
           - AgentlessDSSO
           - X509
         type: string
+        x-okta-known-values:
```

Comment:
I believe this was left for backwards compatibility.

Reply:
@bretterer @laura-rodriguez I feel it would be appropriate to define it as a `type: string` and add `x-okta-known-values` section and delete the below section entirely.

```
type:
        enum:
          - SAML2
          - GOOGLE
          - FACEBOOK
          - LINKEDIN
          - MICROSOFT
          - OIDC
          - OKTA
          - IWA
          - AgentlessDSSO
          - X509
```

I regenerated code from Java SDK now after @sergiishamrai-okta's changes and observed that the generator would treat it as an enum if the above weren't removed. Happy to chat more about the need for this section.

cc: @sergiishamrai-okta